src/node_errors.cc

```diff
@@ -538,10 +539,11 @@ static void ReportFatalException(Environment* env,
       std::string argv0;
       if (!env->argv().empty()) argv0 = env->argv()[0];
       if (argv0.empty()) argv0 = "node";
+      auto filesystem_path = std::filesystem::path(argv0);
       FPrintF(stderr,
               "(Use `%s --trace-uncaught ...` to show where the exception "
               "was thrown)\n",
-              fs::Basename(argv0, ".exe"));
+              filesystem_path.filename().string());
```

Comment:
This no longer appears to remove the `.exe` extension causing `parallel/test-node-output-errors` to fail.

Reply:
Thank you. I forgot to call `path.replace_extension()`. Updating the PR now...